### PR TITLE
added sensu exporter

### PR DESCRIPTION
**Pull request by reachlin**
added [sensu exporter](git push --set-upstream origin sensu-exporter) for prometheus

### CODE REVIEW

**Comment by reachlin** [comment]
@brian-brazil please review my PR. thanks.

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [scollector exporter](https://github.com/tgulacsi/prometheus_scollector)
    * [SNMP exporter](https://github.com/prometheus/snmp_exporter) (**official**)
    * [StatsD exporter](https://github.com/prometheus/statsd_exporter) (**official**)
+   * [Sensu exporter](https://github.com/reachlin/sensu_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Keep it sorted

**Comment by reachlin** [comment]
@brian-brazil thanks for your comments, fixed all but the one about using ConstMetrics. I need more time to understand this, so opened an issue to track: https://github.com/reachlin/sensu_exporter/issues/2

**Comment by reachlin** [comment]
@brian-brazil added customized collector, pls check if this is what you mentioned.

**Comment by brian-brazil** [comment]
Thanks.

That looks right, you don't need that mutex though. Multiple concurrent scrapes are fine.

**Comment by reachlin** [comment]
ok, but confused for multiple places mentioned concurrent issue:
https://github.com/prometheus/haproxy_exporter/blob/master/haproxy_exporter.go
https://github.com/prometheus/client_golang/blob/master/prometheus/example_clustermanager_test.go#L62

**Comment by brian-brazil** [comment]
Not all exporters handle this correctly. A correctly written exporter will automatically handle concurrent scrapes.

**Pull request merged**